externs/react.ext.js

```diff
@@ -0,0 +1,1756 @@
+/**
+ * @type {!Object}
+ * @const
+ */
+var React = {};
+
+/**
+ * @type {string}
+ * @const
+ */
+React.version;
+
+React.createClass = function(specification) {};
+React.createFactory = function(reactClass) {};
```

Comment:
A lot of these are outdated. Once we get the tests passing we should trim these down significantly.